local/docker/wordpress/config/php/php.ini

```diff
@@ -13,7 +13,7 @@ xdebug.mode=off
 xdebug.scream=1
 xdebug.discover_client_host=1
 xdebug.start_with_request=trigger
-
+xdebug.use_compression=0
```

Comment:
Could leave that extra line break after this for visual grouping purposes?

Reply:
Sure thing, I just applied the change!